doc/content_item_fields.md

```diff
@@ -16,7 +16,8 @@ from the content store API can be found in
 
 ## `base_path`
 
-A string. Present in all contexts.
+A string. Present in the request URL in the publishing context, stored as `_id`
```

Comment:
how about skipping "... stored as `_id`"? i feel that's extra information, because it's an implementation detail (mongo-specific). the next paragraph confirms that it is used as a unique identifier, which bit clients would be interested in knowing.